contrib/gn_module_validation/backend/gn_module_validation/blueprint.py

```diff
@@ -282,3 +289,57 @@ def get_validation_date(uuid):
         return jsonify(str(s.last_validation.validation_date))
     else:
         return "", 204
+
+
+# Send notification
+def notifyChange(data, idsynthese):
+
+    # If notification enabled only
+    if current_app.config["NOTIFICATION"]["ENABLED"] == True:
+        nomenclature = TNomenclatures.query.filter(
+            TNomenclatures.id_nomenclature == data["statut"]
+        ).first()
+
+        # idRole is a list separated by coma
+        idRoles = (
+            DB.session.query(Synthese.id_digitiser)
+            .filter(Synthese.id_synthese == int(idsynthese))
+            .one()
+        )
+        roles = [str(role) for role in idRoles]
+
+        # Add mandatory dat
+        categoriesArray = ["VALIDATION-1"]
+        # Statut Certain
+        if data["statut"] == "315":
```

Comment:
```suggestion
        nomenclature.cd_nomenclature == "…":
```
Les ID sont dépendants de la DB

Reply:
Par compris la question. Tu veux utiliser plutôt la mnémonique plutôt que le code de la nomenclature pour la comparaison, c’est ça ? Il n’y a pas de contrainte d’unicité sur la mnémonique contrairement au code, mais c’est plutôt une correction à apporter à la base de données, du coup je vois pas de soucis à ce que utilise la mnémonique de la nomenclature plutôt que le code.